src/reporter/delete.py

```diff
@@ -34,11 +34,19 @@ def delete_entity(entity_id, type_=None, from_date=None, to_date=None):
 def delete_entities(entity_type, from_date=None, to_date=None,
                     drop_table=False):
     with translator_for(fiware_s()) as trans:
-        if drop_table:
```

Comment:
it doesn't look like this change set belongs in this PR? it looks like it's coming from #455?

- https://github.com/orchestracities/ngsi-timeseries-api/pull/455/files

Reply:
should we rollback changes to this file? or merge #455 and then rebase?

Reply:
can you remove changes to this file? thx

Reply:
@daminichopra see my comment above, if you take out this commit from the pr, we will merge it, thx!
if you have trouble doing so, i can open a new pr with the pagination part for you.

Reply:
@chicco785 , Yes my PR has some issues with master branch so not able to remove these commits.  Thanks for suggestion  :)